From 932cf8f1d4f17c38804c0b0a0237d5718bd8d5ac Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E9=BB=84=E6=B6=9B?= <huangtao@rock-chips.com>
Date: Mon, 1 Aug 2011 02:16:52 +0800
Subject: [PATCH] Revert "drivers: misc: pass miscdevice pointer via file
 private data"

This reverts commit fa1f68db6ca7ebb6fc4487ac215bffba06c01c28.
Which broken pmem driver.
---
 drivers/char/misc.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/char/misc.c b/drivers/char/misc.c
index abdafd488980..bfbb2e9f3085 100644
--- a/drivers/char/misc.c
+++ b/drivers/char/misc.c
@@ -144,7 +144,6 @@ static int misc_open(struct inode * inode, struct file * file)
 	old_fops = file->f_op;
 	file->f_op = new_fops;
 	if (file->f_op->open) {
-		file->private_data = c;
 		err=file->f_op->open(inode,file);
 		if (err) {
 			fops_put(file->f_op);
-- 
2.35.3

